R/time.R

```diff
@@ -0,0 +1,197 @@
+#' Time Feature Generator
+#'
+#' `step_time` creates a *specification* of a recipe
```

Comment:
```suggestion
#' `step_time()` creates a *specification* of a recipe
```

I think all new documentation should use the tidyverse standard of writing functions with the `()`